### DISCUSSION TITLE
CC2531 routers suck - simply run two instances of zigbee2mqtt?

### DISCUSSION THREAD

**Discussion by Trashmee** [Koenkk/zigbee2mqtt#6179] [Devices]
Hi,

I've walked down the road for a while, my initial aqara / zigbee2mqtt setup impressed me, so i expanded and used routers for a while. Now I'm using between 10-15 zigbee devices on two floors and my two CC2531 (with antenna) simply annoy me. Cross floor reception sucks. With repeaters, training works mostly, but after a couple of hours / days devices simply disappear forever. I though about buying a different coordinator, but availability, price & unclear outcome keeps me from going there.

Plan B:
- I have a cross floor USB extension, so my docker host has /dev/ttyACM0 + /dev/ttyACM1
- As the configuration samples doesn't mention multiple serials, I'd just spin up two containers, passing ttyACM1 as ttyACM0 for the second container

That seems to work & I'm able to attach devices. 

Open Questions:
- Am I getting something terribly wrong?
- configuration.yaml with 'same floor devices' + permit_join: false is the way to go?
- should i be worried about the mqtt topics? zigbee2mqtt/bridge/networkmap would be subscribed by both and that might be ok, but are there other, more problematic areas?

**Comment by GreenRover** [comment]
Hi i have the same problem with the disappearing devices.
It worked for 1 year with 7 devices.
But since i add 2 more it sucks.
Devices getting lost randomly.
I already updated CC2531 firmeware.
